bottlecap/src/tags/lambda/tags.rs

```diff
@@ -105,6 +106,12 @@ fn tags_from_env(
     if let Ok(memory_size) = std::env::var(MEMORY_SIZE_VAR) {
         tags_map.insert(MEMORY_SIZE_KEY.to_string(), memory_size);
     }
+    let arch = match ARCH {
+        "aarch64" => "arm64",
```

Comment:
I'm not sure if `x86` is going to ever appear, should we: test it until we find it, or remap it either way to `x86` maps to `x86_64`, or naw?

Reply:
x86 would be 32bit.

Here are our current invocations:
<img width="2157" alt="image" src="https://github.com/DataDog/datadog-lambda-extension/assets/1598537/64f64bed-4e85-4e2b-a1a2-a6f16f09c4bb">

Reply:
great tysm for clearing that out